modules/connectivity/ConnectionQuality.js

```diff
@@ -61,32 +34,32 @@ let startBitrate = 800;
  * @param simulcast {boolean} whether simulcast is enabled or not.
  * @param resolution {Resolution} the resolution.
  * @param millisSinceStart {number} the number of milliseconds since sending video started.
- * @param videoQualitySettings {Object} the bitrate and codec settings for the local video source.
+ * @param bitrates {Object} the bitrates for the local video source.
  */
-function getTarget(simulcast, resolution, millisSinceStart, videoQualitySettings) {
+function getTarget(simulcast, resolution, millisSinceStart, bitrates) {
     let target = 0;
     let height = Math.min(resolution.height, resolution.width);
 
     // Find the first format with height no bigger than ours.
-    let simulcastFormat = kSimulcastFormats.find(f => f.height <= height);
+    let qualityLevel = VIDEO_QUALITY_LEVELS.find(f => f.height <= height);
 
-    if (simulcastFormat && simulcast && videoQualitySettings.codec === CodecMimeType.VP8) {
+    if (qualityLevel && simulcast) {
```

Comment:
This reads weird now, because it only applies to VP8, but I don't understand how this condition implies that.

Reply:
Well actually, this doesn't apply to VP8 only anymore. H.264, AV1 and VP9 can also run in simulcast mode (via config overrides) and it is now being correctly detected here https://github.com/jitsi/lib-jitsi-meet/pull/2436/files#diff-c56ee5241e6d324b646a7ec9e618327097f918fba28f70c5396c2ccef69919ecR317 so I fixed this check.